functions/Get-DbaMsdtc.ps1

```diff
@@ -71,17 +71,21 @@ Get-DbaMsdtc -ComputerName srv0042 | Out-Gridview
 
 Get DTC status for the computer srv0042 and show in a grid view
 
+
+
 #>
 	
 	[CmdletBinding()]
 	Param (
-		[Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
-		[string[]]$Computername
+		[string[]]$Computername = $env:COMPUTERNAME
 	)
 	
-	BEGIN
+BEGIN
 	{
+    $functionName = "Get-DbaMsDtc"

```

Comment:
Thanks for this FriedrichWeinmann! @Stuart-Moore, @PowerDBAKlaas and I were discussing it on the Style board. Will add it there.